dev/bots/prepare_package.dart

```diff
@@ -347,7 +347,6 @@ class ArchiveCreator {
   /// is configured for the user to begin working.
   Future<void> _populateCaches() async {
     await _runFlutter(<String>['doctor']);
-    await _runFlutter(<String>['update-packages']);
```

Comment:
update-packages fetches all of the packages of all of our dependencies, which is massively bloating the zip size

Reply:
cc @gspencergoog any reason not to do this?

Reply:
One of the main reasons we created the downloadable archives in the first place was to enable people (mainly in China) to work offline by downloading an archive (on a machine with a connection), putting it on a USB stick, unpacking it on another machine, and being able to run.

If we don't fetch packages, that won't work anymore. If we don't think that is a scenario we care about then that's fine.

We also thought that the out of box experience of unpacking it and building an example was improved because you didn't need to immediately wait for pub to run: it just built (but that's a fairly small thing).

Reply:
we already fetch all of the dependencies for the flutter tool, and anything needed to run a flutter template. Pulling in all of our dependencies, the zip size has gradually increased to 848 MB